1]. DSA + CP/1]. DSA/1]. Data Structures/07]. Doubly Linked List/C++/_02)_InsertAfterGivenNode_DoublyLInkedList.cpp

```diff
@@ -0,0 +1,66 @@
+#include <iostream>
+using namespace std;
+
+//doubly linked list node using struct
+struct Node{
+    int data;
+    Node* next;
+    Node* prev;
+};
+
+//insert node after a given node in doubly linked list
+void insert_after(Node* prev_node, int new_data){
+    //check if the given previous node is NULL
+    if(prev_node == NULL){
+        cout << "The given previous node cannot be NULL";

```

Comment:
Add `new line` after comment print in output
![image](https://user-images.githubusercontent.com/60146338/209458478-ec3dbcc5-34ce-4f74-98e8-691323edf02d.png)

![image](https://user-images.githubusercontent.com/60146338/209458443-5ed62e51-c768-4ab3-bc29-4599985727fc.png)